### DISCUSSION TITLE
Inaccuracy in project documentation

### DISCUSSION THREAD

**Discussion by jekakmail** [blakeblackshear/frigate#14133] [Ask A Question] [answered]
What is the difference between the first and third HTTP API requests? Perhaps it is an inaccuracy in the documentation?
![image](https://github.com/user-attachments/assets/d934d79c-6f61-471a-b08c-4aff999bcac6)
[Documentation](https://docs.frigate.video/integrations/api)

**Comment by NickM-27** [comment]
Looks like the url just wasn't set correctly

**Reply by NickM-27** [reply, marked as answer]
fixed in https://github.com/blakeblackshear/frigate/pull/14136
